### DISCUSSION TITLE
hidden the crousel in smal deviice

### DISCUSSION THREAD

**Discussion by naveenkumarkdn** [tailwindlabs/tailwindcss#13511] [Help]
hidden the crousel in smal deviice

**Comment by alex-krasikau** [comment]
Hey @naveenkumarkdn!

To hide content on small devices, you can use:
```html
<div class="hidden lg:block">
  <Carousel />
</div>

// or

<div class="max-lg:hidden">
  <Carousel />
</div>
```

I recommend the first option for consistent media query direction, but both methods work well.

**Reply by naveenkumarkdn** [reply]
how do fix it javascript also

**Reply by naveenkumarkdn** [reply]
it working but slider in running constantly
